src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Ball {
+    private Colors color;
```

Comment:
Consider using an access modifier for the 'color' field. Without an explicit access modifier, it has package-private access, which might not be intended. If you want to restrict the access, use 'private'.